lib/components/tree-node-collection.component.ts

```diff
@@ -51,10 +48,6 @@ export class TreeNodeCollectionComponent implements OnInit, OnDestroy {
     deprecatedSelector('TreeNodeCollection', 'tree-node-collection', elementRef);
   }
 
-  @action setNodes(nodes) {
```

Comment:
Yup, as I noted on the issue the only change needed to fix the AOT issue is to import `action` from `mobx-angular` instead of `mobx` so you get the metadata for the decorator.